content/components/relative-time.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Relative time
+status: Experimental
+---
+
+import {Box, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  A relative time displays the relative time difference between the source date-time and the provided date-time.
+</Box>
+
+## Content
+
+### Past
+
+For past dates, the relative time outputs different formats depending on the time difference.
+
+Within the current month, the output shows a relative date in the format of `X seconds/minutes/hours/days ago`. If the time difference is less than 1 minute, it shows `just now`.
+
+After a month has passed, the output shows a relative date in the format `on D MMM` or `on D MMM, YYYY` depending on the current year.
+
+<img
+  width="960"
+  src="https://user-images.githubusercontent.com/912236/192233477-8c30b28b-5f48-44c6-b373-39a482f745a1.png"
+  role="presentation"
+/>
+<Caption>A list of items using the past relative outputs.</Caption>
+
+### Future
+
+As well as past dates, future dates output different formats depending on the time difference.
+
+If the future date is due to happen within the current month, the relative time outputs the format of `in X seconds/minutes/hours/days`. If the time difference is less than 1 minute, it shows `just now`.
+
+If the future date is due to happen within the following months, the relative time outputs the format of `on D MMM` or `on D MMM, YYYY` when following years.
+
+### Micro
+
+The `micro` variant shows a smaller version of the relative time. It is useful for displaying the relative time in a more compact space or in a mobile environment. The output shows the relative time in the format of `Xs/h/d`, `D MMM` after a month, or `D MMM, YYYY` after a year.
```

Comment:
```suggestion
The `micro` variant shows a short version of the relative time. It's useful to display the relative time in a more compact space or in a mobile environment. The output shows the relative time in the format of `Xs/h/d` or `D MMM` if the date is not within this month, or `D MMM, YYYY` if the date is not within the current year.
```

Reply:
I rephrased it a bit, as the output is the same for past and future dates + to align with your suggestions

```suggestion
The `micro` format shows a short version of the relative time. It's useful to display the relative time in a more compact space or in a mobile environment. For both past and future dates, the output shows the relative time in the format of `Xs/h/d`. If the date is in the following or past months, the format `D MMM` is used. If the date is in the following or past years, the format `D MMM, YYYY` is used.
```

Reply:
Perfect. 👍